packages/gatsby-plugin-image/README.md

```diff
@@ -117,7 +117,7 @@ If you need to have dynamic images (such as if they are coming from a CMS), you
        body
        avatar {
          childImageSharp {
-           gatsbyImageData(width: 200)
+           gatsbyImage(width: 200)
```

Comment:
I'm not sure we should actually make these changes to this README because

- these are all sub fields on `childImageSharp` fields
- `gatsbyImage` is not supported everywhere
 
It might be better to add new examples showing how to use `gatsbyImage` in addition to a link to a list of source plugins that currently support `gatsbyImage`

@wardpeet @TylerBarnes and thoughts?